examples/pipelines/disk-uploader/README.md

```diff
@@ -0,0 +1,29 @@
+# Disk Uploader Pipeline
+
+This pipeline will deploy a new Virutal Machine (VM) - disk uploader - and then the exported Virutal Machine (VM).
+
+## Prerequisites
+
+- KubeVirt `v0.36.0`
```

Comment:
Have you tested it with so old kubevirt? :D

Reply:
Hehe wrong copy-paste!